### PR TITLE
Added in medicine wheel href to json and formatted file

### PR DESCRIPTION
**Pull request by RussWilkie**
PR for the following issue:

https://github.com/nativesintech/indigemoji/issues/5

### CODE REVIEW

**Comment by RussWilkie** [comment]
I also noticed when I run ` all ` the command breaks at scrapers/wiki-flags/index.js at line 22( `let file_path = $img_a.attr('href').split('/wiki/File:')[1]`). This looks to be happening if `$img_a.attr('href')` is undefined. I put in a conditional to simply do an empty return if it does find the attribute as undefined.

 If preferred, I can remove this commit and just have this PR contain the added Medicine Wheel change.

**Comment by arecvlohe** [comment]
> If preferred, I can remove this commit and just have this PR contain the added Medicine Wheel change.

I think if it helps improve the code then it's a welcome change.

File: scrapers/wiki-flags/index.js [hunk 1]
```diff
@@ -19,6 +19,9 @@ const scrape_wiki = async (url, out_path) => {
     let $elem = $(elem)
     let $img_a = $elem.find('a.image')
 
+    
+    if($img_a.attr('href') === undefined) return
```
Comment (arecvlohe) [inline scrapers/wiki-flags/index.js:23]: In newer node versions I think you can just do `$img_a.attr('href')?.split(...)`, right? Although, I don't know which version of node we are on.

ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Comment (arecvlohe) [reply scrapers/wiki-flags/index.js:23]: Same can apply for the Array index if it doesn't exist I think `?.[1]`

Comment (RussWilkie) [reply scrapers/wiki-flags/index.js:23]: Attempted to give this a go and hit a 404 error. Discovered that is due to the following item in the **data\wiki-flags\us.json** :
 ```
{
    "img_src": "https://www.mediawiki.org/w/index.php?title=Special:Redirect/file/Wyandot_Nation.png",
    "title": "Wyandotte Nation of Oklahoma",
    "slug": "flag-wyandotte-nation-of-oklahoma"
  },
```

Where it looks like the original url in this img_src (https://www.mediawiki.org/w/index.php?title=Special:Redirect/file/Wyandot_Nation.png) no longer has the expected image. When adding the optional chaining it seems like the url is then redefined as `https://www.mediawiki.org/w/index.php?title=Special:Redirect/file/undefined`. 

I was thinking I could add in some additional error handling to output some logging when an undefined is hit. My current commit looks like it'll remove this item once it does the file overwrite. I was wondering if you would have a preference on what to do to this item in the json if the scraper hits an undefined image.

Comment (arecvlohe) [reply scrapers/wiki-flags/index.js:23]: Ah okay, yeah, just keep it as is then

**Pull request merged**